src/lib/pagination/ctrl.coffee

```diff
@@ -0,0 +1,99 @@
+EventsEmitter = require( 'events' )
+Pagination = require( './index' )
+PaginationView = require( './view' )
+
+###
+ * #PaginationCtrl
+ * Creates a pagination element and controls it.
+ *
+ * ## Events
+ *
+ * | CONSTANT.argument  | Description |
+ * |--------------------| -------------|
+ * | CHANGE | When page number is changed by user |
+ * | CHANGE.current  | New page number |
+ * | CHANGE.last | Number of last page |
+###
+class PaginationCtrl extends EventsEmitter
```

Comment:
Your example is not accurate, you're right :)
But I insist: EventEmitter was made for that. Many node classes extend EventEmitter.
I understand what you say but on the other hand, I prefer the class to implement a well know api and avoid repeating the same code over and over again instantiating a class and destroying it. Creating a new dependency "passing it" would be even worse for me.